### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by kubamarkiewicz** [wintercms/wn-blocks-plugin#25] [Ideas]
I was wondering how translations could be implemented. Would it be possible to specify in block definition which fields are translatable? For example if I have two fields in a block, an image and a text, I would like only the text to be translatable, but the image to stay the same for all languages.

Thanks, I love the concept of this plugin.

**Comment by LukeTowers** [comment]
It should be possible, someone would just have to implement it in a PR or pay for me to do it for them 😉 

I would say having a `translatable: true` property on the fields you want to be translatable should cover it.

**Reply by kubamarkiewicz** [reply]
A `translatable` property on the fields would be perfect! I'm happy to hear that this is possible :)
I will see if I can implement it myself or if I find a client willing to pay for this feature than I will contact you.
Thanks!
